themes/hub/layouts/index.json

```diff
@@ -0,0 +1,20 @@
+{{- $scratch := newScratch }}
+{{- $scratch.Add "index" slice }}
+{{- range .RegularPagesRecursive }}
+  {{- if strings.Contains .Permalink "static/" }}{{ continue }}{{ end }}
+  {{- $scratch.Add "tags" slice }}
+  {{- range (.GetTerms "tags") }}
+    {{- $scratch.Add "tags" (dict "title" .LinkTitle "permalink" (printf "/?%s" ((querify "q" .LinkTitle) | safeURL))) }}
```

Comment:
I find that quotes are needed to work the search function as expected when a tag contains "-".

```suggestion
    {{- $scratch.Add "tags" (dict "title" .LinkTitle "permalink" (printf "/?%s" ((querify "q" (printf "\"%s\"" .LinkTitle)) | safeURL))) }}
```

<img width="1354" alt="image" src="https://github.com/optuna/optunahub-web/assets/30489874/808c433a-dd5f-4a03-ae82-8771e8e6fa1f">
<img width="1344" alt="image" src="https://github.com/optuna/optunahub-web/assets/30489874/91d0dd13-3901-40d2-a300-1e4428fe9d80">

Reply:
Setting
```
          ignoreLocation: true,
```
will resolve this issue.